containers-toolkit/Public/AllToolsUtilities.psm1

```diff
@@ -72,7 +72,12 @@ function Install-ContainerTools {
     )
 
     begin {
-        $toInstall = @("containerd", "buildkit", "nerdctl")
+        # Strip leading "v" from version
+        $containerdVersion = $containerdVersion.TrimStart("v")
```

Comment:
so basically the `v` will be optional, and it can work with or without, right? LGTM.